deep-learning/_posts/2017-03-27-layer-normalization.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: review
+title: Layer Normalization
+tags: deep-learning essentials 
```

Comment:
Please add the tag `layers`. (The review of Batch Norm should have it too, I thought it had it)